--- lib/python2.7/site-packages/txAMQP-0.4-py2.7.egg/txamqp/codec.py.ori	2011-06-21 18:18:43.282532427 +0200
+++ lib/python2.7/site-packages/txAMQP-0.4-py2.7.egg/txamqp/codec.py	2011-06-21 18:19:16.902699147 +0200
@@ -187,6 +187,8 @@
         value = self.decode_longstr()
       elif type == "I":
         value = self.decode_long()
+      elif type == "F":
+        value = self.decode_table()
       else:
         raise ValueError(repr(type))
       result[key] = value
